src/common/app_metrics.cpp

```diff
@@ -352,4 +354,9 @@ void buddy::metrics::record_dwarf_internal_temperatures() {
         }
     }
 }
+
+void buddy::metrics::RecordUptime() {
+    METRIC_DEF(uptime, "uptime", METRIC_VALUE_INTEGER, 1000, METRIC_HANDLER_ENABLE_ALL);
```

Comment:
The "1000" there means that we would be sending the metric every second. I would say that is a bit wasteful.